Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:2.7-stretch
+
+WORKDIR /app
+
+RUN groupadd --gid 10001 app \
+    && useradd -g app --uid 10001 -s /usr/sbin/nologin app
+
+
+COPY ./dev-requirements.txt /app/dev-requirements.txt
+
+# Copy in the whole app after dependencies have been installed & cached
```

Comment:
What's this comment about? We don't do anything with dependencies before this `COPY` command.